modules/host.go

```diff
@@ -104,6 +104,9 @@ type (
 		// have been made to the host.
 		NetworkMetrics() HostNetworkMetrics
 
+		// PublicKey returns the public key of the host.
+		PublicKey() types.SiaPublicKey
```

Comment:
you may expect this function to be in the HostExternalSettings but we explicitly chose to have it be separate because we didn't want the renter code accidentally reading off a new pubkey and thinking it was the same host (which would be a nasty way to bypass Sybil attack defenses we have)